src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -77,12 +74,14 @@ export const ArtworkSidebarCommercialButtons: React.FC<ArtworkSidebarCommercialB
       extractNodes(me.partnerOffersConnection)[0]) ||
     null
 
-  // Fall back to a definitely past value because the timer hook doesn't like nulls
-  const partnerOfferTimer = useTimer(partnerOffer?.endAt || THE_PAST)
   const partnerIcon = artwork.partner?.profile?.icon?.url
 
-  const activePartnerOffer =
-    (!partnerOfferTimer.hasEnded && partnerOffer) || null
+  const isPartnerOfferPast = useMemo(() => {
+    if (!partnerOffer?.endAt) return false
+    return new Date(partnerOffer.endAt) < new Date()
+  }, [partnerOffer])
+
+  const activePartnerOffer = (!isPartnerOfferPast && partnerOffer) || null
```

Comment:
Looking at this now: why would we fallback to `null`...

Reply:
Oh wow — it's not a boolean that's why. So very confusing but I'll leave it alone.